README.md

```diff
@@ -19,16 +19,11 @@
 Plone backend [Docker](https://docker.com) images using Python 3 and [pip](https://pip.pypa.io/en/stable/).
 
 > **NOTE**:
```

Comment:
`>` is a block quote or a quotation of a paragraph. I would remove all of these from the README. Also the colon should have the same formatting as the word NOTE.

```suggestion
**Note:**
```